apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/model/bp/BarnepensjonInnvilgelse.kt

```diff
@@ -116,6 +117,7 @@ data class BarnepensjonInnvilgelseRedigerbartUtfall(
                 erEtterbetaling = etterbetaling != null,
                 harFlereUtbetalingsperioder = utbetalingsinfo.beregningsperioder.size > 1,
                 erGjenoppretting = generellBrevData.systemkilde == Vedtaksloesning.GJENOPPRETTA,
+                harUtbetaling = beregningsperioder.any { it.utbetaltBeloep.value > 0 },
```

Comment:
Ser vi to rader over heller bruker utbetalingsinfo.beregningsperioder, har det noko å seie?